src/GeoPing.Api/Models/AccountViewModels/RegisterViewModel.cs

```diff
@@ -8,6 +8,11 @@ namespace GeoPing.Api.Models.AccountViewModels
 {
     public class RegisterViewModel
     {
+        [Required]
```

Comment:
this is for MVC Helpers, we don't need them

Reply:
done